packages/api/src/lib/structures/Route.ts

```diff
@@ -36,6 +42,7 @@ export abstract class Route extends Piece {
 		}
 
 		this.maximumBodyLength = options.maximumBodyLength ?? api.maximumBodyLength ?? 1024 * 1024 * 50;
```

Comment:
Same as with my prior comment. Or, as this is in multiple places, perhaps declare a constants enum for things like this? Up to you, ultimately. Just a suggestion.

Reply:
The reason why this is inlined is because declaring a constant or an enum just for this would add clutter.

Besides... who says V8 doesn't optimise inline math operations? :P